crates/storage/provider/src/bundle_state/bundle_state_with_receipts.rs

```diff
@@ -38,16 +38,18 @@ pub struct BundleStateWithReceipts {
 // TODO(mattsse): unify the types, currently there's a cyclic dependency between
 impl From<BatchBlockExecutionOutput> for BundleStateWithReceipts {
     fn from(value: BatchBlockExecutionOutput) -> Self {
-        let BatchBlockExecutionOutput { bundle, receipts, first_block } = value;
+        let BatchBlockExecutionOutput { bundle, receipts, requests: _, first_block } = value;
         Self { bundle, receipts, first_block }
     }
 }
 
 // TODO(mattsse): unify the types, currently there's a cyclic dependency between
+#[cfg(any(test, feature = "test-utils"))]
```

Comment:
we still need this I believe

Reply:
yeah but it doesn't include the requests in the bundle state now, so for now I gated it behind the cfg and added a todo